bemani/backend/popn/usaneko.py

```diff
@@ -160,11 +160,12 @@ def __construct_common_info(self, root: Node) -> None:
             13: 1,
         }
 
-        for phaseid in phases:
+    def __construct_common_info(self, root: Node) -> None:
+        for phaseid in self.get_phases():
```

Comment:
Suggest `for phaseid, phase in self.get_phases().items():` here so that you don't have to call `self.get_phases()` repeatedly in the loop below. This will be come a bigger deal when event structures get hooked up to the arcade config screen.